connect/runtime/src/test/java/org/apache/kafka/connect/integration/ConnectorClientPolicyIntegrationTest.java

```diff
@@ -133,7 +133,7 @@ private void assertFailCreateConnector(String policy, Map<String, String> props)
             connect.configureConnector(CONNECTOR_NAME, props);
             fail("Shouldn't be able to create connector");
         } catch (ConnectRestException e) {
-            assertEquals(e.statusCode(), 400);
+            assertEquals(400, e.statusCode());
```

Comment:
The expected value should go first, as it's used in error messages that come up with test failures ("expected <first argument>, got <second argument>").

This test did fail once locally for me; I believe it was just some flakiness induced by my laptop being overworked.